addons/website_hr_recruitment/__manifest__.py

```diff
@@ -20,6 +20,7 @@
     'demo': [
         'data/hr_job_demo.xml',
     ],
+    'auto_install': True,
```

Comment:
a reason?

Reply:
I think it seems interesting to have website_hr_recruitment if you have all the dependencies as you'd probably want to promote the jobs offers on your website

Reply:
Not my pov, but why not @sbuhl ?

My idea is that you can handle your CV and recurtment process, without specially want to have form online to submit your cv. But btw since it is uninstallable if you don't want it... I'm open to all.

Reply:
Same POV as JKE on this one. With the logic "if the user has that, he probably wants that", we can make all apps auto_install :wink:

Reply:
And if you want to use your website for that, just hit the "new" button and you will have a button to add job offers and this will install website_hr_recruitment for you.